zerver/views/__init__.py

```diff
@@ -120,7 +120,7 @@ def accounts_register(request):
         realm = get_realm(domain)
 
     if realm_creation and not settings.OPEN_REALM_CREATION:
-        return render_to_response("zerver/realm_creation_disabled.html")
+        return render_to_response("zerver/realm_creation_failed.html", {'message': 'This server no more allow members of the public to create new organizations.'})
```

Comment:
You need to do the `_()` here (where the actual error string is defined), not in the template, for Django's translation string extraction to work correctly.